.github/workflows/release.yml

```
@@ -40,28 +21,40 @@ jobs:
             image: ubuntu:24.04
           - name: fedora39
             image: fedora:39
-    runs-on: ubuntu-24.04
+          - runner: archlinux
```

should this be `name` instead of `runner`?
```suggestion
          - name: archlinux
```

---

Oop, nice catch